lib/models/project.js

```diff
@@ -14,9 +14,14 @@ const versionUtils = require('../utilities/version-utils');
 const emberCLIVersion = versionUtils.emberCLIVersion;
 const findAddonByName = require('../utilities/find-addon-by-name');
 const heimdall = require('heimdalljs');
-const PackageInfoCache = require('../models/package-info-cache');
+const PackageInfoCache = require('./package-info-cache');
+const PerBundleAddonCache = require('./per-bundle-addon-cache');
 
-const instantiateAddons = require('../models/instantiate-addons');
+const instantiateAddons = require('./instantiate-addons');
+
+// Support for per-bundle addon caching is opt OUT. Caching any particular
+// addon class is currently opt IN.
+const SUPPORT_ADDON_INSTANCE_CACHING = process.env.EMBER_CLI_ADDON_INSTANCE_CACHING !== 'false';
```

Comment:
Let's make `per-bundle-addon-cache/index.js` the "source of truth" for this value, and import it from there.

Reply:
much better - thanks.

Reply:
Done.